packages/commonwealth/server/routes/contracts/createContract.ts

```diff
@@ -73,24 +85,24 @@ const createContract = async (
     throw new AppError(Errors.NotAdmin);
   }
 
-  if (!contractType || !contractType.trim()) {
-    throw new AppError(Errors.NoType);
-  }
+  // if (!contractType || !contractType.trim()) {
+  //   throw new AppError(Errors.NoType);
+  // }
 
   const Web3 = (await import('web3-utils')).default;
   if (!Web3.isAddress(address)) {
     throw new AppError(Errors.InvalidAddress);
   }
 
-  if (decimals < 0 || decimals > 18) {
-    throw new AppError(Errors.InvalidDecimal);
-  }
+  // if (decimals < 0 || decimals > 18) {
+  //   throw new AppError(Errors.InvalidDecimal);
+  // }
   if (!chain_node_id) {
     throw new AppError(Errors.NoNodeUrl);
   }
-  if (!balance_type) {
-    throw new AppError(Errors.InvalidBalanceType);
-  }
+  // if (!balance_type) {
+  //   throw new AppError(Errors.InvalidBalanceType);
+  // }
 
   let abiAsRecord: Array<Record<string, unknown>>;
   if (abi) {
```

Comment:
@alexyoung23j also here below, on the UI we don't have nickname for abi in this case. But below there is a check that breaks the flow if the there is ABI included